vic/drivers/shared_image/src/vic_start.c

```diff
@@ -99,18 +99,14 @@ vic_start(void)
         }
 
         // get dimensions (number of vegetation types, soil zones, etc)
-        options.ROOT_ZONES = get_nc_dimension(filenames.soil, "root_zone");
-        options.Nlayer = get_nc_dimension(filenames.soil, "nlayer");
-        options.NVEGTYPES = get_nc_dimension(filenames.veg, "veg_class");
-        if (options.SNOW_BAND > 1) {
-            if (options.SNOW_BAND !=
-                get_nc_dimension(filenames.snowband, "snow_band")) {
-                log_err("Number of snow bands in global file does not "
-                        "match parameter file");
-            }
+        options.ROOT_ZONES = get_nc_dimension(filenames.params, "root_zone");
+        options.Nlayer = get_nc_dimension(filenames.params, "nlayer");
+        options.NVEGTYPES = get_nc_dimension(filenames.params, "veg_class");
+        if (options.SNOW_BAND > SNOW_BAND_UNSET) {
```

Comment:
I'm confused about how the `SNOW_BAND_UNSET` parameter works. In vic/drivers/image/src/get_global_param.c line 486, `options.SNOW_BAND = SNOW_BAND_UNSET` if the SNOW_BAND option in the global file is `TRUE`. How does `options.SNOW_BAND` end up `> SNOW_BAND_UNSET`?